README.md

```diff
@@ -52,7 +52,7 @@ QA no longer needs help to test your services. Enable them to autonomously confi
 
 #### ⛵️ Container management across environments
 
-Whether you're a developer or an indie hacker, bring your own infrastructure and manage all of your containers and applications from one place while our software is running either as a SaaS or on your infrastructure.
+Whether you're a developer or an indie hacker, please bring your own infrastructure and manage all your containers and applications from one place while our software is running either as a SaaS or on your infrastructure.
```

Comment:
In my opinion this `please` is not required, because we aren't asking our users to bring their own infra, we just want to tell them that they can use whatever they want.

All the other modifications are cool 😎. Thanks.

Reply:
thankyou too sir @robot9706